journey/src/androidTest/kotlin/com/pingidentity/journey/callback/recaptchaEnterprise/RecaptchaEnterpriseCallbackTest.kt

```diff
@@ -0,0 +1,668 @@
+     * 5. Asserts output message contains expected verification details
+     *
+     * **Assertions:**
+     * - Site key matches expected configuration
+     * - Token is successfully generated and non-empty
+     * - Payload contains correct token, action (LOGIN), and empty client error
+     * - Output message includes valid risk score (0.0-1.0), token properties, and Android package name
+     * - Action in response matches LOGIN action
+     */
+    @Test
+    fun testRecaptchaEnterpriseSuccess() = runTest {
+        var node = recaptchaJourney.start(tree) as ContinueNode
+
+        node.handleLoginCallbacks()
+        node = node.next() as ContinueNode
+
+        val choiceCallback = node.callbacks.first() as ChoiceCallback
+        choiceCallback.selectedIndex = choiceCallback.choices.indexOf("success")
+        node = node.next() as ContinueNode
+
+        val recaptchaEnterpriseSuccess = node.callbacks.first() as ReCaptchaEnterpriseCallback
+        assertTrue(recaptchaEnterpriseSuccess.reCaptchaSiteKey.isNotEmpty())
+        assertEquals(
+            RECAPTCHA_SITE_KEY,
+            recaptchaEnterpriseSuccess.reCaptchaSiteKey
+        )
+        var tokenFromCallbackResult = ""
+
+        runBlocking {
+            recaptchaEnterpriseSuccess.verify{}.onSuccess { token ->
+                assertTrue(token.isNotEmpty())
+                tokenFromCallbackResult = token
+            }.onFailure {
+                assertTrue(false) // Should not reach here
+            }
+        }
+        val inputPayload = recaptchaEnterpriseSuccess.payload()["input"]
+        assertNotNull(inputPayload)
+        inputPayload?.jsonArray?.forEach {
+            when (it.jsonObject["name"]?.toString()?.replace("\"", "")) {
+                "IDToken1token" -> {
+                    val tokenFromPayload = it.jsonObject["value"]?.toString()?.replace("\"", "") ?: ""
+                    assertEquals(tokenFromCallbackResult, tokenFromPayload)
+                }
+                "IDToken1action" -> {
+                    val action = it.jsonObject["value"]?.toString()?.replace("\"", "") ?: ""
+                    assertEquals(action, RecaptchaAction.LOGIN.action)
+                }
+                "IDToken1clientError" -> {
+                    val clientError = it.jsonObject["value"]?.toString()?.replace("\"", "") ?: ""
+                    assertTrue(clientError.isEmpty())
+                }
+                "IDToken1payload" -> {
+                    val payload = it.jsonObject["value"]?.toString()?.replace("\"", "") ?: ""
+                    assertEquals("{}", payload)
+                }
+            }
+        }
+
+        node = node.next() as ContinueNode
+        val textOutputCallback = node.callbacks.first() as TextOutputCallback
+        val outputMessageJson = Json.parseToJsonElement(textOutputCallback.message)
+        assertNotNull(outputMessageJson)
+
+        val name = outputMessageJson.jsonObject["name"]?.jsonPrimitive
```

Comment:
.jsonObject may returns null

<img width="2384" height="878" alt="Image" src="https://github.com/user-attachments/assets/3c9046f9-c805-4582-8cf2-cdcc74eb3c15" />

Reply:
Please review other area having similar issue in this test.